skiko/src/commonTest/kotlin/org/jetbrains/skia/FontMgrTest.kt

```diff
@@ -94,6 +95,7 @@ class FontMgrTest {
 
     @Test
     @SkipJsTarget
+    @SkipNativeTarget
```

Comment:
It only shall be disabled on iOS, if at all. See https://github.com/JetBrains/skiko/blob/09aad6044e04a0b87ae4a9489de9c2236417d735/skiko/src/iosTest/kotlin/org/jetbrains/skiko/Utils.kt#L12